addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -220,6 +266,27 @@ export class SearchBar extends Component {
         return subItems;
     }
 
+    /**
+     * Compute the properties items, only when we unfold the properties field
+     * (will fetch the properties definition on the parent to generate the items).
+     *
+     * @param {Object} searchItem
+     * @param {Object} field
+     */
+    async _getPropertiesSearchItems(searchItem, field) {
```

Comment:
Would not name it "_getPropertiesSearchItems" as it does not return anything.

Maybe "_insertPropertiesSearchItems" ?

Also I don't think we need 2 separate methods for this, you could just do everything here at once, it's not likely to get re-used.
(Or maybe you have plans to re-use the method of the searchModel elsewhere?).